test cases/unit/69 static link/test1.c

```diff
@@ -0,0 +1,6 @@
+int func2();
+
+int main(int argc, char *argv[])
+{
+  return func2() == 2 ? 0 : 1;
```

Comment:
This test should be a bit more elaborate. There should be a `func3` also in file `func1.c` and the main function should call it directly. This checks that the linking work both when the function is used directly and indirectly. For `link_whole` both of these should work.

Reply:
Ok, added that and it works :)